bindings/python/mainloop.pyx

```diff
@@ -0,0 +1,17 @@
+
+cdef extern from "sol-mainloop.h":
+	int sol_init()
+	int sol_run()
+	void sol_shutdown()
+
+class Mainloop():
+	def __init__(self):
+		sol_init()
+
+	def __del__(self):
+		sol_shutdown()
+
+	def run(self):
+			return sol_run()
```

Comment:
Maybe take the `startup` and `shutdown` functions so we simulate the behavior as https://github.com/solettaproject/soletta/blob/master/src/lib/common/include/soletta.h#L105?

The idea is that we run user code inside the main loop. This guarantees `sol_init()` is called and mainloop is running. As said above, auto-calling `sol_init()` on module load may be a solution that solves that issue.